app/controllers/detailedvaluationrequest/DvrController.scala

```diff
@@ -311,43 +314,51 @@ class DvrController @Inject()(
         authorisationId: Long,
         uarn: Long,
         isOwner: Boolean): Action[AnyContent] = authenticated.async { implicit request =>
-    propertyLinks.canChallenge(plSubmissionId, assessmentRef, caseRef, isOwner) flatMap {
-      case None =>
-        Future.successful {
-          val returnUrl =
-            if (isOwner)
-              s"${config.serviceUrl}${controllers.detailedvaluationrequest.routes.DvrController.myOrganisationRequestDetailValuationCheck(plSubmissionId, assessmentRef, uarn).url}"
-            else
-              s"${config.serviceUrl}${controllers.detailedvaluationrequest.routes.DvrController.myClientsRequestDetailValuationCheck(plSubmissionId, assessmentRef, uarn).url}"
-          Redirect(
-            config.businessRatesValuationFrontendUrl(s"property-link/valuations/startChallenge?backLinkUrl=$returnUrl"))
-        }
-      case Some(response) =>
-        if (response.result) {
-          val party = if (isOwner) "client" else "agent"
-          Future.successful(
-            Redirect(config.businessRatesChallengeUrl(
+    val eventualPropertyHistory: Future[PropertyHistory] = vmvConnector.getPropertyHistory(uarn)
```

Comment:
I can see there have been changes to what the controller does, however I don't find an accompanying change in the unit/integration tests. Can you add a unit test for this controller method?